modules/distribution/src/assembly/bin.xml

```diff
@@ -909,119 +879,98 @@
+            </source>
             <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/</outputDirectory>
             <filtered>true</filtered>
         </file>
 
-        <!--email broker config-->
+        <!-- Copying identity-mgt.properties -->
+        <!--file>
+            <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/identity-mgt.properties</source>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/identity</outputDirectory>
+            <filtered>true</filtered>
+        </file-->
+
+        <!-- Copying email-admin-config.xml -->
+        <!--file>
+            <source>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/email/email-admin-config.xml</source>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/email</outputDirectory>
+            <filtered>true</filtered>
+        </file-->
+
+        <!-- Copying axis2-client.xml -->
         <file>
-            <source>src/repository/conf/axis2/axis2_client.xml</source>
-            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/axis2/
+            <source>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/axis2/axis2_client.xml
+            </source>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/axis2
             </outputDirectory>
             <filtered>true</filtered>
             <fileMode>644</fileMode>
         </file>
+
+        <!-- Copying axis2.xml -->
         <file>
             <source>src/repository/conf/axis2/axis2.xml</source>
-            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/axis2/</outputDirectory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/axis2
+            </outputDirectory>
             <filtered>true</filtered>
             <fileMode>644</fileMode>
         </file>
 
+
         <file>
-            <!--<source>src/repository/conf/etc/launch.ini</source>-->
-            <source>target/wso2carbon-core-${carbon.kernel.version}/repository/conf/etc/launch.ini</source>
+            <source>src/repository/conf/etc/launch.ini</source>
             <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/etc/
             </outputDirectory>
             <filtered>true</filtered>
             <fileMode>644</fileMode>
         </file>
 
+        <!-- Copying MDM jaggery app -->
+        <!--<file>-->
+        <!--<source>../apps/jaggery/mdm/target/mdm.zip</source>-->
+        <!--<outputDirectory>${pom.artifactId}-${pom.version}/repository/deployment/server/jaggeryapps/-->
+        <!--</outputDirectory>-->
+        <!--<filtered>false</filtered>-->
+        <!--<fileMode>644</fileMode>-->
+        <!--</file>-->
+
+        <!-- Copying authenticators.xml -->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.